ports/servoshell/desktop/headless_window.rs

```diff
@@ -171,4 +171,9 @@ impl PlatformWindow for Window {
             self.screen_size.height as u32,
         ));
     }
+
+    fn show_console_message(&self, level: servo::ConsoleLogLevel, message: &str) {
+        println!("{}", message);
+        log::log!(level.into(), "{}", message);
```

Comment:
I think you can omit this.
```suggestion
        log::log!(level.into(), "{message}");
```